src/App.tsx

```diff
@@ -58,19 +59,22 @@ const defaultFilters: SearchFilterType = {
   is_collecting_samples: false,
 };
 
-interface GeolocationCoordinates {
-  coords: {
-    latitude: number;
-    longitude: number;
-    accuracy: number;
-  };
-  timestamp: number;
+class LatLng {
+  lat: number;
+  lng: number;
+
+  constructor(lat: number, lng: number) {
+    this.lat = lat;
+    this.lng = lng;
+  }
 }
```

Comment:
This seems unnecessary. Line 116 can just do `{ lat, lng }` instead and you get the same result